.gitignore

```diff
@@ -0,0 +1,15 @@
+# IntelliJ project files
```

Comment:
Check that one https://github.com/Andrei-Khadziukou/petclinic-order/pull/8/files. Which do you like more?
@susau, which one do you prefer?